src/man/firejail-profile.txt

```diff
@@ -467,6 +467,12 @@ available in the new namespace is a new loopback interface (lo).
 Use this option to deny network access to programs that don't
 really need network access.
 
+.SH Other
+.TP
+\fBjoin-or-start sandboxname
+Join the sandbox identified by name or start a new one.
+Same as "firejail --join=sandboxname" command if sandbox with specified name exists, otherwise same as "name sandboxname".
```

Comment:
Oh, okay. I wasn't thinking of that.
It is perhaps a bit confusing (since there is no `join` option for the profiles) but I would leave it like you have it.

Oops!